src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -663,10 +677,20 @@ protected Model createCleanPom( Model effectivePom ) throws MojoExecutionExcepti
             {
                 if ( !isEmpty( profile.getDependencies() ) || !isEmpty( profile.getRepositories() ) )
                 {
+                    List<Dependency> strippedDependencies = new ArrayList<>();
+                    for (Dependency dep : profile.getDependencies()) {
+                        Dependency flattenedDep = createFlattenedDependency( dep );
+                        if ( flattenedDep != null )
+                        {
+                            strippedDependencies.add( flattenedDep );
+                        }
+                    }
+                    if (strippedDependencies.isEmpty() && isEmpty(profile.getRepositories()))
```

Comment:
it may be better to maintain the same style as the original code, where as instead of the continue, which can be easily missed, wrap the logic in the if block like before.

Reply:
update: use curly brackets instead of continue